puppetboard/app.py

```diff
@@ -362,16 +384,22 @@ def reports(env, page):
     """
     check_env(env)
 
+    if env == '*':
+        reports_query = None
+        total_query = '["extract", [["function", "count"]]]'
```

Comment:
Only PuppetDB 3.2 supports the count function WITHOUT a query expression. My first implementation of this was like this:

``'["extract", [["function", "count"]], ["~", "certname", ""]]'``